services/debugger/cache/internal/badger/badger.go

```diff
@@ -27,25 +27,27 @@ func (e *Cache[E]) loadCacheConfig() {
 	config.RegisterDurationConfigVariable(5, &e.ticker, false, time.Minute, "LiveEvent.cache.GCTime")
```

Comment:
shouldn't we decrease this too, e.g. to `10s`?

Reply:
Should we reduce this even on our systems? Everything looks fine for us right so I just left this as is

Reply:
Is there any particular reason to make GC more frequent? 

The increased frequency will hardly affect the used storage, while it might become less efficient. 

I suggest increasing the frequency if we observe long GC pauses.

Reply:
The issue might occur in our higher-traffic deployments, e.g. `bolt`. `ValueLogGC` doesn't cause the system to pause. Since in this use case we are overwriting the same key multiple times within `5m`, it would be better to perform cleanup in higher frequencies.

Reply:
I stand corrected:  ~long GC pauses~ -> GC execution times

Within the 10s period, there might not be a log page to garbage collect and there is certainly some cost involved in calling the GC more frequently. 

We can do a quick test for `bolt` and see the impact of this change. 

I understand that this is minor, but given how important badgerDB is for us, I would like to make an informed decision about this change and increase our understanding.